### ISSUE TITLE
Camera not opening

### ISSUE THREAD

**Issue by arthtilva** [card-io/card.io-Android-SDK#52]
When I tried the same code in Eclipse Project it does not open Camera in real and virtual device both,  it is giving a CardDetail activity.How can i resolve this?
![capture](https://cloud.githubusercontent.com/assets/7376106/6796772/b9c3c6da-d21e-11e4-8242-b2b0ea469356.JPG)

**Comment by braebot** [comment]
Can you please supply the logs when you try to open the camera?  Not all devices are supported.  Virtual devices are complicated to get right, and so may have issues as well.

**Comment by arthtilva** [comment]
![capture](https://cloud.githubusercontent.com/assets/7376106/6798306/db3b3880-d231-11e4-9010-bbfdc61d69e2.JPG)

**Comment by arthtilva** [comment]
![capture](https://cloud.githubusercontent.com/assets/7376106/6798411/bf40310c-d232-11e4-83e7-d20efd6c6b65.JPG)

**Comment by arthtilva** [comment]
Also provide a way to check whether device is supported or not...

**Comment by arthtilva** [comment]
I tried and saw log it was showing device not supported, and another thing I want to ask that whether it is available for Eclipse or not?

**Comment by kraghu** [comment]
@braebot I am facing the same Issue. This is the error I am seeing , 
03-24 19:07:38.389  14360-14360/com.android.consumer E/card.io﹕ Failed to load native library: Couldn't load cardioDecider from loader dalvik.system.PathClassLoader[dexPath=/data/app/com.android.consumer-223.apk,libraryPath=/data/app-lib/com.android.consumer-223]: findLibrary returned null

What library is it expecting ? I might be missing .so library as I couldnt get studio working with maven .So I downloaded Jar file from the .aar file. Please help me out ? Am using Android studio 1.1.0 and ran on samsung s5 


Is

**Comment by braebot** [comment]
Are you all able to get the sample app working?  It uses Gradle / Android Studio, and should work right out of the box.  If you use eclipse, you must add the processor specific libs and directories.  Please see https://github.com/card-io/card.io-Android-SDK/issues/15 for more information on this same issue.

**Comment by kraghu** [comment]
@braebot  I am using android studio. For some reason I am not able to access maven central repository al though links seems to be fine. I ended up downloading aar file and changing the build files. If it helps , this is what I did .
- I created libs folder and downloaded .aar file 
- tell Studio to look for libs folder in build.cradle (the higher level where you can see all repositories)
allprojects {
    repositories {
              jcenter()
                   flatDir {
                       dirs 'libs'
                }
       }
 }
- Now add this line in module build.cradle file under dependencies
compile 'io.card:android-sdk:5.0.0@aar'

Notice the name android-sdk instead of card.io from the instructions. Thats where I was wrong.

**Comment by braebot** [comment]
@kraghu, so does your solution work?  If so, what line of the build instructions are wrong?  We'd love to correct them if there's an issue.

**Comment by kraghu** [comment]
@braebot  In your instruction you have this line ,
     If you use gradle, then add the following dependency from mavenCentral():
     compile 'io.card:android-sdk:5.0.0'
This would work if mavencentral() is accessible from studio. For some weird reason maven repository is unaccesible. So What I did , 
 - I downloaded cardio.aar and placed it inside the apps/libs folder
-  Studio expects .aar or .zip files to be inside libs/folder. In a build.cradle file (module:app with the one "apply plugin:" is on the top) Under dependencies add this line,
                      compile 'io.card:android-sdk:5.0.0@aar'

This worked for me.
